influxdb.rb

```diff
@@ -64,7 +64,11 @@ def run(event_data)
         key.gsub!("\\"){ "\\\\" }
 
         # This will merge : default conf tags < check embedded tags < client tags < sensu client/host tag
-        tags = @influx_conf['tags'].merge(event[:check][:influxdb][:tags]).merge(event[:client][:tags]).merge({'host' => client})
+        if event[:client].has_key(:tags)
```

Comment:
This is erroring for me, should be has_key? with ? right?

Reply:
I think Zac is right, since the function returns a boolean:
```if event[:client].has_key?(:tags)```
http://ruby-doc.org/core-2.2.1/Hash.html#method-i-has_key-3F